torch/nn/utils/fusion.py

```diff
@@ -16,6 +16,10 @@ def fuse_conv_bn_eval(conv, bn):
 def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
     if conv_b is None:
         conv_b = bn_rm.new_zeros(bn_rm.shape)
+    if bn_w is None:
+        bn_w = torch.ones(bn_rm.shape)
```

Comment:
should it instead be `torch.ones_like(bn_rm)` to preserve device? or at least `torch.ones(bn_rm.shape, device = bn_rm.device)`?